packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -92,7 +95,31 @@ export const ChatMessageComponentAsRichTextEditBox = (
     onCancelInlineImageUpload
   } = props;
 
-  const [textValue, setTextValue] = useState<string>(message.content || '');
+  const initialContent = useMemo(() => {
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    const content = message.content;
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    const document = new DOMParser().parseFromString(content ?? '', 'text/html');
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    document.querySelectorAll('.broken-image-wrapper').forEach((brokenImage) => {
+      const imageElement = document.createElement('img');
+      const attributes = brokenImage.attributes;
+      for (const attribute of attributes) {
+        // if (attribute.name !== 'data-ui-id' && attribute.name !== 'class') {
+        imageElement.setAttribute(attribute.name, attribute.value);
+        // }
+      }
+
+      imageElement.src = BROKEN_IMAGE_SVG_DATA;
```

Comment:
Can we just update src instead of creating a new img tag?

Reply:
Added some comment on this. The broken image is a div element and we need to convert it to img element for RoosterJs to render it.